rdeps.xml

```diff
@@ -9,10 +9,7 @@
   </os>
   <os names="all">
     <pip>
-      <pkg name="PyYAML">pyyaml</pkg>
-      <pkg name="six">six</pkg>
-      <pkg name="wheel">wheel</pkg>
-      <pkg name="virtualenv">virtualenv</pkg>
```

Comment:
Yes I think that all of these are only needed for venv/Tensile, should be fine to remove.